bigbluebutton-html5/imports/ui/components/dropdown/styles.scss

```diff
@@ -230,6 +230,11 @@ $dropdown-caret-height: 8px;
 .top-left {
   @extend %down-caret;
   @extend %horz-right-caret;
+  min-width: 11.5rem;
+
+  @include mq($small-only) {
```

Comment:
I added a media query for making min-width is not working on the mobile view, because if I set a fixed size for dropdown list, on the mobile view, the dropdown list is not filling the whole screen.